morsecode.rb

```diff
@@ -0,0 +1,55 @@
+# rubocop:disable Metrics/MethodLength
+
+def decode(char)
+  check = {
+    'a' => '.-',
+    'b' => '-...',
+    'c' => '-.-.',
+    'd' => '-..',
+    'e' => '.',
+    'f' => '..-.',
+    'g' => '--.',
+    'h' => '....',
+    'i' => '..',
+    'j' => '.---',
+    'k' => '-.-',
+    'l' => '.-..',
+    'm' => '--',
+    'n' => '-.',
+    'o' => '---',
+    'p' => '.--.',
+    'q' => '--.-',
+    'r' => '.-.',
+    's' => '...',
+    't' => '-',
+    'u' => '..-',
+    'v' => '...-',
+    'w' => '.--',
+    'x' => '-..-',
+    'y' => '-.--',
+    'z' => '--..',
+    ' ' => ' '
+  }
+
+  check.key(char).upcase
+end
```

Comment:
- [ ] When I run the other tests, they all pass but the one to print "MY NAME" does not seem to be working. It throws an arrow. See the screenshot below. Kindly fix the error it's returning 👍 so everything works fine as expected.

![image](https://user-images.githubusercontent.com/50941074/199934494-a329c4a2-b9d6-4864-89cb-6035290fb377.png)

Reply:
Thank you for the review